src/components/kytos/table/Table.vue

```diff
@@ -56,9 +61,105 @@ export default {
         return []
       }
     },
+  },
+  data() {
+    return {
+      currentPage: 1,
+      pageSize: 5,
+      pages: []
+    }
+  },
+  methods: {
+    activeCurrentPage() {
+      /**
+       * Highlight the current page indicator
+       */
+      let currPages = $('.page-number')
+
+      for (let i = 0; i < currPages.length; i++) {
+        (currPages[i].id.includes(this.currentPage)) ?
+            $(currPages[i]).addClass('active') : $(currPages[i]).removeClass('active')
+      }
+    },
+    nextPage() {
+      /**
+       * Move pagination forward
+       */
+      ((this.currentPage * this.pageSize) < this.rows.length) ?
+        this.currentPage++ : this.currentPage = 1
```

Comment:
If the one-liner spans two lines, maybe it should be a standard if/else. ;)

Reply:
I really have a thing with this kind of if/else ahahhahaha sorry!